cli/dcoscli/node/main.py

```diff
@@ -748,20 +750,25 @@ def _ssh(leader, slave, option, config_file, user, master_proxy, command):
     if command is None:
         command = ''
 
-    master_public_ip = dcos_client.metadata().get('PUBLIC_IPV4')
-    if master_proxy:
+    proxy_ip = proxy_ip or config.get_config_val('core.dcos_proxy_ip')
```

Comment:
Thanks. I know it's convenient for that command, but I don't want to add config/envvars that only apply to one command when they could apply to others. It just causes lots of confusion. Feel free to create a JIRA or github issue to add this option globally (and have it used globally).